ui/src/view/grid/row.rs

```diff
@@ -41,9 +42,12 @@ impl<T> Row<T> {
         let mut element: El<T> = El::empty(Tag::Custom(Cow::Borrowed("row")));
 
         if self.center {
-            element.add_class("center");
+            let center_class = "center";
+            element.add_class(center_class);
         }
 
+        style::apply(vec![style::padding(4)], &mut element);
```

Comment:
So here is an example of some styling api. To apply padding, you can use `style.padding(4)`, which is a function that returns a `Style`. 

This style api stuff isnt done yet, because theres no real way to apply `Style`s yet. Theres just this hardcoded instance of all `row` getting a padding of `4`.